authors/apps/articles/models.py

```diff
@@ -232,3 +232,27 @@ def _update_article_tags(self, instance, new_tag_list):
             # Add the tag to the article instance.
             instance.tags.add(the_tag)
         return True
+
+
+class ReportArticle(models.Model):
+    """ model to hold report instances """
+
+    article = models.ForeignKey(
+        Article,
+        to_field='slug',
+        on_delete=models.CASCADE
+    )
+    message = models.CharField(
+        max_length=225, null=False
+    )
+    reporter = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='reports',
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self):
+        return self.message
+
+    def get_username(self):
+        return self.reporter.username
```

Comment:
@BrianSerem, I don't seem to see how this method comes into play in the PR. I can't see how you are picking the reporter as username anywhere, kindly clarify on this.

Reply:
This is a method used in the serializer to get the username of the reporter.